pkgs/dkg/drand.go

```diff
@@ -81,29 +93,53 @@ func (msg *Result) Decode(data []byte) error {
 	return json.Unmarshal(data, msg)
 }
 
+type PriShare struct {
+	I int    `json:"index"`
+	V []byte `json:"secret_point"`
+}
+
+type DistKeyShare struct {
+	Commits []byte   `json:"commits"`
+	Share   PriShare `json:"secret_share"`
+}
+
+// Encode returns a msg encoded bytes or error
+func (msg *DistKeyShare) Encode() ([]byte, error) {
+	return json.Marshal(msg)
+}
+
+// Decode returns error if decoding failed
+func (msg *DistKeyShare) Decode(data []byte) error {
+	return json.Unmarshal(data, msg)
+}
+
 var ErrAlreadyExists = errors.New("duplicate message")
 
+// LocalOwner as a main structure created for a new DKG initiation or resharing ceremony
 type LocalOwner struct {
 	Logger      *zap.Logger
-	startedDKG  chan struct{}
+	StartedDKG  chan struct{}
 	ErrorChan   chan error
 	ID          uint64
-	data        *DKGData
-	b           *board.Board
-	suite       pairing.Suite
+	Data        *DKGData
+	Board       *board.Board
+	Suite       pairing.Suite
```

Comment:
why all of these are public now?, not a big deal but I'm not sure there's a good reason.

Reply:
I was trying to use it at another modules. Just left it exported eventually

Reply:
wanted to use it, but not. Left exported.

Reply:
Anything exported should be commented about how it is used. if it is not used and not meant to be used. unexport it. if it is meant to be used add a comment about how it is used. this is about everything exported in this package.